scripts/update/jellyfin.sh

```diff
@@ -0,0 +1,80 @@
+        [[ -d "/home/${username}/.config/Jellyfin/root" ]] && cp -fRT "/home/${username}/.config/Jellyfin/root" /var/lib/jellyfin/root
+        #
+        sed -r 's#<PublicPort>(.*)</PublicPort>#<PublicPort>8096</PublicPort>#g' -i /etc/jellyfin/system.xml
+        sed -r 's#<PublicHttpsPort>(.*)</PublicHttpsPort>#<PublicHttpsPort>8920</PublicHttpsPort>#g' -i /etc/jellyfin/system.xml
+        sed -r 's#<HttpServerPortNumber>(.*)</HttpServerPortNumber>#<HttpServerPortNumber>8096</HttpServerPortNumber>#g' -i /etc/jellyfin/system.xml
+        sed -r 's#<HttpsPortNumber>(.*)</HttpsPortNumber>#<HttpsPortNumber>8920</HttpsPortNumber>#g' -i /etc/jellyfin/system.xml
+        sed -r 's#<RequireHttps>false</RequireHttps>#<RequireHttps>true</RequireHttps>#g' -i /etc/jellyfin/system.xml
+        #
+        if [[ -f /etc/nginx/apps/jellyfin.conf ]]; then
+            sed -r 's#proxy_pass https://127.0.0.1:(.*);#proxy_pass https://127.0.0.1:8920;#g' -i /etc/nginx/apps/jellyfin.conf
+            systemctl -q restart nginx.service
+        fi
+        #
+        rm_if_exists "/home/${username}/.config/Jellyfin"
+        rm_if_exists "/home/${username}/.cache/jellyfin"
+        rm_if_exists "/home/${username}/.aspnet"
+    fi
+    #
+    if ! check_installed jellyfin; then
+        echo_info "Updating Jellyfin"
+        #
+        # Add the jellyfin official repository and key to our installation so we can use apt-get to install it jellyfin and jellyfin-ffmepg.
+        wget -q -O - "https://repo.jellyfin.org/$DIST_ID/jellyfin_team.gpg.key" | apt-key add - >> "${log}" 2>&1
+        echo "deb [arch=$( dpkg --print-architecture )] https://repo.jellyfin.org/$DIST_ID $DIST_CODENAME main" > /etc/apt/sources.list.d/jellyfin.list
+        #
+        # install jellyfin and jellyfin-ffmepg using apt functions.
+        apt_update #forces apt refresh
+        apt_install jellyfin jellyfin-ffmpeg
+        #
+        # Configure the new jellyfin service.
+        systemctl -q stop jellyfin.service
+        #
+        # Add the jellyfin user to the master user's group to use our ssl certs.
+        usermod -a -G "${username}" jellyfin
+        #
+        # Set the correct and required permissions of any directories we created or modified.
+        chown "${username}.${username}" -R "/home/${username}/.ssl"
+        chmod -R g+r "/home/${username}/.ssl"
+        #
+        # Set the default permissions after we have migrated our data
+        chown -R jellyfin:jellyfin "/etc/jellyfin"
+        chown jellyfin:root "/etc/jellyfin/logging.json"
+        chown jellyfin:adm "/etc/jellyfin"
+        chown -R jellyfin:jellyfin "/var/lib/jellyfin"
+        chown jellyfin:adm "/var/lib/jellyfin"
+        #
+        # Reload systemd and start the service.
+        systemctl -q daemon-reload
+        systemctl -q start jellyfin.service
+        #
+        echo_success "The Jellyfin upgrade has completed"
```

Comment:
Should be "Jellyfin installed" to fit the pattern of the others

Reply:
And in this event I'd even use "Jellyfin updated" actually

Reply:
Changed to `Jellyfin updated`